handbook/engineering/README.md

```diff
@@ -632,6 +632,8 @@ If the bug does not meet the criteria of a critical bug, the EM will determine i
 
 When fixing the bug, if the proposed solution requires changes that would affect the user experience (UI, API, or CLI), notify the EM and PM to align on the acceptability of the change.
 
+Engineering teams collaboratively handle bug fixes. If one team is at capacity and a bug needs attention, another team can step in to assist. When the MDM team helps with a CX bug, use the `~leaning on MDM` label to ensure the bug shows on the #g-mdm board. If the CX team helps with an MDM bug, use the `~leaning on CX` label to ensure the bug shows on the #g-cx board.
```

Comment:
We'll need to adjust slightly to work with ZenHub's label filtering. For example, if a CX bug goes to the MDM team, we will take the following steps: 

1. Remove the `#g-cx` label (this will remove it from the CX sprint board) 
2. Add the `#g-mdm` label (this will add it to the MDM board) 
3. Add a `~assisting CX` label so the MDM team understands where it came from. 

So we need the labels to be `~assisting CX` or `~assisting MDM`.